app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    arguments = command.split()
+    if len(arguments) != 3:
+        raise ValueError("Wrong count of arguments")
+    com, original_filename, copy_filename = arguments
+    if com != "cp":
+        raise ValueError("Wrong command")
+    with (open(arguments[1], "r") as file_in,
+          open(arguments[2], "w") as file_out):
+        if original_filename != copy_filename:
```

Comment:
It would be better to place this check in the if condition on line 6 to not open files without necessity. Place this check earlier.